nautobot/extras/tests/test_jobs.py

```diff
@@ -731,3 +731,28 @@ def test_remove_stale_scheduled_jobs(self):
         self.assertTrue(ScheduledJob.objects.filter(name="test1").exists())
         for i in range(2, 7):
             self.assertFalse(ScheduledJob.objects.filter(name=f"test{i}").exists())
+
+
+class ScheduledJobIntervalTestCase(TestCase):
+    """Test scheduled job intervals"""
+
+    # cron schedule day_of_week starts on Sunday (Sunday = 0)
+    cron_days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+    # datetime weekday starts on Monday (Sunday = 6)
+    datetime_days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
```

Comment:
I know this might seem pedantic, but I hate seeing hard-coded data like this.

What if:

```python
import calendar

sunday_calendar = calendar.Calendar(calendar.SUNDAY)
cron_days = [calendar.day_name[i] for i in sunday_calendar.iterweekdays()]
datetime_days = list(calendar.day_name)
```

The rest would remain the same. And yes I'm fun at parties.

Reply:
My first iteration was:

```py
# cron week starts on Sunday (Sunday = 0)
# python datetime week starts on Monday (Sunday = 6)
scheduled_day_of_week = list(scheduled_job.schedule.day_of_week)[0]
datetime_weekday_to_cron_weekday = (start_time.weekday() + 1) % 7
self.assertEqual(scheduled_day_of_week, datetime_weekday_to_cron_weekday)
```

I put the static lists in there for maintainability but I can change it. Thoughts on the block above?

Reply:
Eh, honestly no worries. The static lists are a solved problem and easier to read comparatively.